### DISCUSSION TITLE
Substraction of big double

### DISCUSSION THREAD

**Discussion by Perpelle** [Estia-advanced-programming/pandora-public#82] [Q&A] [answered]
Hello, i have an issue with the calcul of windSpeed, when I calculate the difference between a timestamp and the previous one i have 0.0 in result. I try to calculate at first the distance between every value (by using longitude and latitude and haversine) and then the same with the time (with substraction of timestamp) but it just give me 0.0 as a result even though the first and the second timestamp are different.

**Comment by dhmmasson** [comment, marked as answer]
Hello, create a pull request and add me as a reviewer 
Comment in the files changed so that I can look into it : 
https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request

Also you are free to save data however you want in the Record object.

**Comment by Perpelle** [comment]
I made the pull request and added you as reviewer

**Comment by dhmmasson** [comment]
I let this one open, so other students with the same issue may find it more easily
